shell/browser/ui/inspectable_web_contents_impl.cc

```diff
@@ -456,6 +458,10 @@ void InspectableWebContentsImpl::ShowDevTools(bool activate) {
     managed_devtools_web_contents_ = content::WebContents::Create(
         content::WebContents::CreateParams(web_contents_->GetBrowserContext()));
     managed_devtools_web_contents_->SetDelegate(this);
+    v8::Isolate* isolate = v8::Isolate::GetCurrent();
+    v8::HandleScope scope(isolate);
+    api::WebContents::FromOrCreate(isolate,
```

Comment:
For devtools, we already create the js wrapper in `api::WebContents::DevToolsOpened`

Reply:
That wrapper is not created early enough to respond to the sync message that is sent when the renderer boots.